parsec/backend/postgresql/realm_queries/get.py

```diff
@@ -89,6 +122,38 @@ async def query_get_status(
     )
 
 
+@query()
+async def query_get_stats(
+    conn, organization_id: OrganizationID, author: DeviceID, realm_id: UUID
+) -> RealmStats:
+    ret = await conn.fetchrow(
+        *_q_has_realm_access(
+            organization_id=organization_id, realm_id=realm_id, user_id=author.user_id
+        )
+    )
+    if not ret:
+        raise RealmNotFoundError(f"Realm `{realm_id}` doesn't exist")
+
+    if not ret["has_access"]:
+        raise RealmAccessError()
+    blocks = await conn.fetch(
+        *_q_get_blocks_size_from_realm(organization_id=organization_id, realm_id=realm_id)
+    )
+    vlobs = await conn.fetch(
+        *_q_get_vlob_size_from_realm(organization_id=organization_id, realm_id=realm_id)
+    )
+    RealmStats.blocks_size = 0
+    RealmStats.vlobs_size = 0
+
+    for block in blocks:
```

Comment:
I helped @Sithi5 write this code, but I didn't realized it was for the sql part... It seems highly inefficient to me, and we should use sql SUM instead, WDYT @touilleMan ?

Reply:
I updated my sql part to use SUM instead. It seems indeed more appropriate

Reply:
Seems perfect! You can add more complex UT or make the existing ones more complete if you want, for example creating multiple `vlobs` or `blocks`, but I'll accept the PR as is :)